examples/scripts/cycling_ageing.py

```diff
@@ -30,24 +30,28 @@
             "Hold at 4.2 V until C/20",
             "Rest for 30 minutes",
             "Discharge at C/3 until 2.8 V",
+            "Rest for 30 minutes",
         ),
         (
             "Charge at 1 C until 4.2 V",
             "Hold at 4.2 V until C/20",
             "Rest for 30 minutes",
             "Discharge at 1 C until 2.8 V",
+            "Rest for 30 minutes",
         ),
         (
             "Charge at 1 C until 4.2 V",
             "Hold at 4.2 V until C/20",
             "Rest for 30 minutes",
             "Discharge at 2 C until 2.8 V",
+            "Rest for 30 minutes",
         ),
         (
             "Charge at 1 C until 4.2 V",
             "Hold at 4.2 V until C/20",
             "Rest for 30 minutes",
-            "Discharge at 3 C until 2.8 V",
+            "Discharge at 3 C until 2.8 V (10 second period)",
+            "Rest for 30 minutes",
```

Comment:
Had to add the rest periods and reduce the sampling period at discharge, otherwise simulations didn't converge.